src/rockstor/storageadmin/views/netatalk.py

```diff
@@ -122,8 +122,9 @@ def post(self, request):
         try:
             for share in shares:
                 mnt_pt = ('%s%s' % (settings.MNT_PT, share.name))
-                cur_description = '%s %s' % (share.name, description)
```

Comment:
From having a look at the original code and your proposed change, Your changes work. So that's a plus. But they do change the original logic which looks like it was intended to allow for the addition of a user input description, and if not found would default to this Share.name + " on Rockstor" which has proved to be contentious. That is share.name is never == '' or if it is we have other problems :).
I think for the time being a better approach to this issue would be to only apply your line 125 changes and additionally:
change line 95 from:

    def_description = 'on Rockstor'

to

    def_description = ''

I.e take out the contents between the 2 single inverted commas. That way the rest of the optional description handling stuff can await further attention on the UI front but we still end up with dropping the " on Rockstor" bit. It's a bit hacky but should do for now to force change our default AFP export name to that of the share.

Reply:
Good idea is to add description field for every share name.

Reply:
Yes, it would be good to add this feature to the UI in the future, especially given the backing code is already done.